cmd/bootstrap/run/epochs.go

```diff
@@ -135,6 +147,8 @@ func GenerateRecoverEpochTxArgs(log zerolog.Logger,
 	}
 
 	args := []cadence.Value{
```

Comment:
The target end time and duration values [are for the recovery epoch](https://github.com/onflow/flow-core-contracts/blob/feature/efm-recovery/contracts/epochs/FlowEpoch.cdc#L642-L643), _not_ the current epoch. 

Good catch. We need to change this. I would suggest the following:
- Keep the single duration flag, but rename it to `epoch-timing-recovery-duration`. Since recovery epochs may be unusual lengths, using a different name discourages re-using a flag value from the bootstrapping logic, which has a similar flag.
- Compute TargetEndTime as `currentEpoch.TargetEndTime() + timingDurationFlag`

Reply:
[`2168811` (#6424)](https://github.com/onflow/flow-go/pull/6424/commits/2168811874396ea8651765cb1f2adcbe6f86032c)

Reply:
The implementation, but it still seems to be inconsistent to me: see by comments below 
👉 https://github.com/onflow/flow-go/pull/6424#discussion_r1843137480
👉 https://github.com/onflow/flow-go/pull/6424#discussion_r1843142014